quiz.py

```diff
@@ -1,6 +1,65 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Garren Yu
+Date: 2024-03-01
+Description: a simple multiple-choice quiz game
+"""
+#header
+print("Multiple-Choice Quiz Game")
+
+#variable to storage total score
+total_score = 0
+
+#print Question 1 and get answer from user
+print("\n1. longest length unit\
+      \n(a) 12in\
+      \n(b) 24.5cm\
+      \n(c) 0.00013mi")
+question_one = input("\n")
```

Comment:
I see you're using separate variables to track user input. It doesn't appear that you do anything with the three variables at the end of the program — in other words, why do you need to remember what the user entered for questions 1, 2, 3, etc.? You can make the program more memory efficient by using only one variable for user input, then reassigning (reusing) it every new question.